modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx

```diff
@@ -79,6 +79,10 @@ const CustomView: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 				objectDefinitionExternalReferenceCode
 			);
 
+			const objectDefinition = await API.getObjectDefinitionById(
+				objectDefinitionId
+			);
+
```

Comment:
Hey @alinedoleron, As of this [Epic here](https://issues.liferay.com/browse/LPS-164800), we are now using the ERC endpoints to do all GET on Object Definitions, so could you change this endpoint to `getObject Definition ExternalReferenceCode()`?